pkg/engine/engine.go

```diff
@@ -102,14 +105,32 @@ var warnRegex = regexp.MustCompile(warnStartDelim + `(.*)` + warnEndDelim)
 func warnWrap(warn string) string {
 	return warnStartDelim + warn + warnEndDelim
 }
+func goid() int {
+	var buf [64]byte
```

Comment:
so if I'm reading this right... If the stack exceeds 64 goroutines we will return a panic stack trace. That seems like a major regression. Can you clarify the behaviour here?

There are several larger charts that will very easily hit this limit with their usage of the `tpl` function.